app/views/users/index.html.erb

```diff
@@ -1,14 +1,18 @@
-<p style="color: green"><%= notice %></p>
+ <main>
+        <section class="section1">
```

Comment:
Good work implementing `Devise` for a better UX 

- Kindly make your sign_in page default when you open your app, in this way, the user can sign in to your application if he isn't already logged